docs/cloud/recipes/k8s_dask.md

```diff
@@ -67,16 +67,16 @@ spec:
               dask-scheduler:8786,
               --no-bokeh,
               --nthreads,
-              "1",
+              '1',
               --nprocs,
-              "2",
+              '2',
             ]
           name: dask-worker
           env:
             - name: DASK_DISTRIBUTED__SCHEDULER__BLOCKED_HANDLERS
               value: "['feed', 'run_function']"
             - name: DASK_DISTRIBUTED__SCHEDULER__WORK_STEALING
-              value: "False"
+              value: 'False'
```

Comment:
are we sure the single quotes are valid here?

Reply:
Ugh, autoformatting. Believe they do work, but I'll change them back for consistency with the rest of our docs.

Reply:
@cicdw Reverted.